### PR TITLE
Add update tags reusable workflow

### PR DESCRIPTION
**Pull request by nelscata**
Adding a workflow that will update the image tag in the values yaml file to allow ArgoCD syncing the new version of the app that is being updated

### CODE REVIEW

File: .github/workflows/cd-update-tag.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        repository: ${{ inputs.repository }}
+        token: ${{ secrets.token }}
+
+    - name: Install yq
+      id: setup-yq
+      uses: shiipou/setup-yq-action@v2.2.0
+
+    - name: Update Tags
+      run: |
+        # change tag with yq
```
Comment (jfmrm) [inline .github/workflows/cd-update-tag.yml:36]: we need here to add the yq command that edits the image tag. Example:
```
yq -i ".global.image.tag = \"$tag\"" <file-path>
```

**Review by jfmrm — CHANGES_REQUESTED** [review]

File: .github/workflows/cd-update-tag.yml (hunk 1 above)
Comment (nelscata) [reply .github/workflows/cd-update-tag.yml:39]: done, added the missing command, in addition an input var to get the path of the values file was added

File: .github/workflows/cd-update-tag.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      service:
+        required: true
+        type: string
+        description: "Service that will be updated with new image tag"
+      tag:
+        required: true
+        type: string
+        description: "Image tag"
+      values-path:
+        required: false
```
Comment (jfmrm) [inline .github/workflows/cd-update-tag.yml:18]: this should be required.

**Review by jfmrm — APPROVED** [review]

File: .github/workflows/cd-update-tag.yml (hunk 2 above)
Comment (nelscata) [reply .github/workflows/cd-update-tag.yml:21]: updated it

**Pull request merged**